source/src/main/resources/templates/bigquery/insert.sqlt

```diff
@@ -0,0 +1,21 @@
+INSERT INTO [[${table.schema}]].[[${table.name}]]
+(
+    [# th:each="column : ${table.columns}"]
+        [[${column.name}]]
+
+        [# th:if="${columnStat.index < table.columns.size - 1}"],[/]
+    [/]
+)
+VALUES
+(
+    [# th:each="column : ${table.columns}"]
+        [# th:switch="${column.typeName}"]
```

Comment:
This is one recommended approach to determine how we want to display the INSERT statement for each column type, using the switch statement.